interfaces/Glitter/templates/include_overlays.tmpl

```diff
@@ -210,6 +210,10 @@
                                         <span class="glyphicon glyphicon-ok"></span> <span data-bind="text: serversslinfo"></span>
                                     </div>
                                 </div>
+                                <div class="row">
+                                    <div class="col-sm-6">$T('speed')</div>
+                                    <div class="col-sm-6" data-bind="text: serverbps"></div>
+                                </div>
```

Comment:
Let's add the speed below the `# Connections`.

Reply:
I've already changed it. However, now it will come between the # connections and the list of connections if displaying those is enabled.